File: Java/SivitasAkademik.java
public class SivitasAkademik extends Human {
    private String asal_universitas, email_edu;

    public SivitasAkademik() {
        this.asal_universitas = "";
        this.email_edu = "";
    }

    public SivitasAkademik(String asal_universitas, String email_edu) {
        this.asal_universitas = asal_universitas;
        this.email_edu = email_edu;
    }

    public SivitasAkademik(String nik, String nama, char jenis_kelamin, String asal_universitas, String email_edu) {
        super(nik, nama, jenis_kelamin);
        this.asal_universitas = asal_universitas;
        this.email_edu = email_edu;
    }

    public void setAsalUniv(String asal_universitas) {
        this.asal_universitas = asal_universitas;
    }

    public void setEmailUniv(String email_universitas) {
        this.email_edu = email_universitas;
    }

    public String getAsalUniv() {
        return this.asal_universitas;
    }

    public String getEmailUniv() {
        return this.email_edu;
    }
}

File: PHP/SivitasAkademik.php
<?php
require "Human.php";

class SivitasAkademik extends Human {
    private $asal_universitas = "";
    private $email_edu = "";
    
    public function __construct($nik = "", $nama = "", $jk = '', $asalUniv = "", $emailUniv = "") {
        parent::__construct($nik, $nama, $jk);
        $this->asal_universitas = $asalUniv;
        $this->email_edu = $emailUniv;
    }

    public function setAsalUniv($asalUniv) {
        $this->asal_universitas = $asalUniv;
    }

    public function setEmailUniv($emailUniv) {
        $this->email_edu = $emailUniv;
    }

    public function getAsalUniv() {
        return $this->asal_universitas;
    }

    public function getEmailUniv() {
        return $this->email_edu;
    }
}
?>